templates/typescript/server.ts

```diff
@@ -36,25 +46,30 @@ const routes = {
   {{/if}}
 };
 
-_.map(routes, (handler: VoxaPlatform, route: string) => {
-  expressApp.post(
-    route,
-    async (
-      req: express.Request,
-      res: express.Response,
-      next: express.NextFunction
-    ) => {
-      try {
-        console.log(chalk.cyan(JSON.stringify(req.body, null, 2)));
-        const reply = await handler.execute(req.body);
+if (config.server.hostSkill) {
+  _.map(routes, (handler: VoxaPlatform, route: string) => {
+    expressApp.post(
+      route,
+      async (
+        req: express.Request,
+        res: express.Response,
+        next: express.NextFunction
+      ) => {
+        try {
+          console.log(chalk.cyan(JSON.stringify(req.body, null, 2)));
+          const reply = await handler.execute(req.body);
 
-        console.log(chalk.green(JSON.stringify(reply, null, 2)));
-        res.json(reply);
-      } catch (e) {
-        next(e);
+          console.log(chalk.green(JSON.stringify(reply, null, 2)));
```

Comment:
Let's try to use lambda logs, console.log are not well formatted on cloudwatch

Reply:
Actually those console logs shouldn't be there lol, I'll remove them